infra/k8s/README.md

```diff
@@ -59,7 +73,7 @@ export ZONES=eu-central-1a
 export KOPS_STATE_STORE=s3://kops-backend-bucket
 export WORKER_NODES=4
 export CLUSTER_SPEC=~/cluster.yaml
-export PUBKEY=~/.ssh/id_rsa.pub
+export PUBKEY=~/.ssh/testground_rsa.pub
```

Comment:
I am still keeping this option, because members from the community should be able to use Testground as well, and they don't want to be getting our shared key.